components/ArticleReply.js

```diff
@@ -175,8 +175,8 @@ class ArticleReply extends React.PureComponent {
     const originalAuthorElem = (
       <EditorName
         key="editorName"
-        editorName={replyAuthor.name}
-        editorLevel={replyAuthor.level}
+        editorName={replyAuthor?.name}
```

Comment:
Does js have optional syntax as typescript does now?

Reply:
We are using https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining
It's currently in stage 3: https://github.com/tc39/proposal-optional-chaining